samples/v1.0/Scenarios/InputForm.json

```diff
@@ -31,35 +31,25 @@
 							"wrap": true,
 							"size": "small"
 						},
-						{
-							"type": "TextBlock",
-							"text": "Your name",
-							"wrap": true
-						},
 						{
 							"type": "Input.Text",
 							"id": "myName",
+							"label": "Your name",
```

Comment:
Does the version number need to be updated? The `label` attribute seems to be working fine without updating the version number but I'm not sure if we should still update it for consistency?

Reply:
the card version is... poorly named. it's effectively the minimum version required to display the card, so leaving it as-is is perfectly fine for this case 👍